apps/src/code-studio/components/progress/StageLockDialog.jsx

```diff
@@ -0,0 +1,284 @@
+import React from 'react';
+import Radium from 'radium';
+import { connect } from 'react-redux';
+import BaseDialog from '@cdo/apps/templates/BaseDialog';
```

Comment:
Oh right. I got confused myself. I forgot that clicking the button will give you the immediate feedback of changing the table, but then you still need to click Save to save the changes of the table. Perhaps in the instructions we should say "don't forget to save your changes each time" or something like that. Because I just had a momentary blip thinking that clicking the action button was all I had to do.